doc/api/esm.md

```diff
@@ -437,10 +437,34 @@ The `--experimental-json-modules` flag is needed for the module
 to work.
 
 ```bash
-node --experimental-modules --entry-type=module index.js # fails
-node --experimental-modules --entry-type=module --experimental-json-modules index.js # works
+node --experimental-modules index.mjs # fails
+node --experimental-modules --experimental-json-modules index.mjs # works
 ```
 
+## Experimental WASM Modules
+
+Importing Web Assembly modules is supported under the
+`--experimental-wasm-modules` flag, allowing any `.wasm` files to be
+imported as normal modules while also supporting their importing of JS modules.
```

Comment:
Are their imports limited to JS modules or can they also import other WASM modules ?

Reply:
WASM modules can import other WASM modules fine too. I'll clarify this line.